README.md

```diff
@@ -167,7 +167,7 @@ export GEMINI_API_KEY='yourkey'
 ```shell
 ./gateway discover \
   --ai-provider gemini \
-  --config connection.yaml \
+  --connection-string connection.yaml \
```

Comment:
`--connection-string ` accept not a file, but a connection string dsn, something like:

```
postgresql://neondb_owner:MY_PASSWORD@MY_HOST.neon.tech/neondb?sslmode=require
```